### PR TITLE
Next Job documentation Update

### PR DESCRIPTION
**Pull request by rehrenfeld**

### CODE REVIEW

File: NextJob.md [hunk 1]
```diff
@@ -47,17 +80,24 @@ for more information about carotene classification and carotene IDs.
...
         "carotene_id": "11.0",
         "carotene_title": "sales manager (management)",
+        "carotene_description": "Plan, direct, or coordinate the actual distribution or movement of a product or service to the customer. Coordinate sales distribution by establishing sales territories, quotas, and goals and establish training programs for sales representatives. Analyze sales statistics gathered by staff to determine sales potential and inventory requirements and monitor the preferences of customers.",
+        "salary": {
+          "currency": "USD",
+          "percentile_10": 10.0,
+          "percentile_90": 14.0,
+          "period": "YEAR"
+        },
+       "active_job_count": 0,
```
Comment (glpayson) [inline NextJob.md]:
```suggestion
        "salary": {
          "currency": "USD",
          "percentile_10": 45000.0,
          "percentile_90": 110000.0,
          "period": "YEAR"
        },
       "active_job_count": 561,
```

Comment (glpayson) [reply NextJob.md]: Let's give this slightly more realistic values.

File: NextJob.md [hunk 2]
```diff
@@ -94,6 +134,15 @@ as well as a list of `common_skills` and list of `missing_skills`.
...
                 "carotene_title": "business development manager (management)",
+                "carotene_description": "Plan, direct, or coordinate marketing policies and programs, such as determining the demand for products and services offered by a firm and its competitors, and identify potential customers. Develop pricing strategies with the goal of maximizing the firm's profits or share of the market while ensuring the firm's customers are satisfied. Oversee product development or monitor trends that indicate the need for new products and services.",
+                "confidence": 0.016686638076838227,
+                "salary": {
+                  "currency": "USD",
+                  "percentile_10": 10.0,
+                  "percentile_90": 14.0,
+                  "period": "YEAR"
+                },
+                "active_job_count": 12,
```
Comment (glpayson) [inline NextJob.md]:
```suggestion
                  "currency": "USD",
                  "percentile_10": 52000.0,
                  "percentile_90": 134000.0,
                  "period": "YEAR"
                },
                "active_job_count": 128,
```

File: NextJob.md [hunk 3]
```diff
@@ -47,17 +80,24 @@ for more information about carotene classification and carotene IDs.
 
 ## Response Structure
 
-The response consist of a `carotene_id`, `carotene_title`, and list of `skills` for the requested 
+The response consist of a `carotene_id`, `carotene_title`, `carotene_description`, `salary`,`active_job_count`  and list of `skills` for the requested 
 `carotene_id`. Each item in the `skills` list contains `term`, `id` strings, and `description`
```
Comment (glpayson) [inline NextJob.md]: Skills now have types.

File: NextJob.md [hunk 4]
```diff
@@ -47,17 +80,24 @@ for more information about carotene classification and carotene IDs.
 
 ## Response Structure
 
-The response consist of a `carotene_id`, `carotene_title`, and list of `skills` for the requested 
+The response consist of a `carotene_id`, `carotene_title`, `carotene_description`, `salary`,`active_job_count`  and list of `skills` for the requested 
 `carotene_id`. Each item in the `skills` list contains `term`, `id` strings, and `description` 
 strings as well as  list of links to relevant videos. The `next_jobs` sections is 
-a list consisting of objects containing a `carotene_id` and `carotene_title` string for the next job
-as well as a list of `common_skills` and list of `missing_skills`.
+a list consisting of objects containing a `carotene_id`, `carotene_title`, `carotene_description`, associated `confidence` to carotene, `salary`,`active_job_count` and  string for the next job as well as a list of `common_skills` and list of `missing_skills`.
```
Comment (glpayson) [inline NextJob.md]:
```suggestion
a list consisting of objects containing a `carotene_id`, `carotene_title`, `carotene_description`, `confidence`, `salary`,`active_job_count` and  string for the next job as well as a list of `common_skills` and list of `missing_skills`.
```

File: NextJob.md [hunk 5]
```diff
@@ -13,32 +13,65 @@ _______
 ## Summary
 
 The Next Job service returns common next jobs for a given carotene ID. For example, Administrative 
-Assistant and Office Manager are common next jobs for Receptionists. Additionally it returns the 
+Assistant and Office Manager are common next jobs for Receptionists. Additionally it returns carotene description and the 
 set of skills associated with the requested carotene ID, as well as the skills the carotene ID has
 in common with its next jobs and the missing skills a person should acquire to pursue the next job. 
+Moreover, for each job we provide active jobs count based on location passed to the endpoint.
+Salary will be provided based on criteria :
```
Comment (glpayson) [inline NextJob.md]:
```suggestion
Salary will be provided based on the following criteria:
```

File: NextJob.md (hunk 4 above)
Comment (rehrenfeld) [reply NextJob.md]: fixed

**Comment by MalarCB** [comment]
@glpayson  Still v1 is in use, don't we want to create NextJobV2.md?

**Comment by glpayson** [comment]
> @glpayson Still v1 is in use, don't we want to create NextJobV2.md?

I'd rather not tbh. We typically treat earlier versions as deprecated and don't keep old documentation around for it. I guess if we really wanted to we could link to the older version of the documentation [like this](https://github.com/careerbuilder/DataScienceAPIDocumentation/blob/f2def45ffb6d80f25043d6060b07b0537c04f5d7/NextJob.md).

**Review by glpayson — APPROVED** [review]

**Review by yuantao-yin — APPROVED** [review]

**Review by ashraj10c1 — APPROVED** [review]

**Review by DavidGWu — APPROVED** [review]

**Pull request merged**